modules/programs/git.nix

```
@@ -296,16 +296,75 @@ in {
           '';
         };
       };
+
+      diff-so-fancy = {
+        enable = mkEnableOption "" // {
+          description = ''
+            Enable the <command>diff-so-fancy</command> diff colorizer.
+            See <link xlink:href="https://github.com/so-fancy/diff-so-fancy" />.
+          '';
+        };
+        markEmptyLines = mkOption {
```

Please make sure to separate option definitions by an empty line to make the module more readable. That is,
```suggestion

        markEmptyLines = mkOption {
```